test/sdk/e2e/get-tickmap.test.ts

```diff
@@ -0,0 +1,225 @@
+import { ONE_ALPH, sleep, web3 } from '@alephium/web3'
+import { getSigner } from '@alephium/web3-test'
+import { Invariant } from '../../../src/invariant'
+import { Network } from '../../../src/network'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+import { getBasicFeeTickSpacing } from '../../../src/snippets'
+import { TokenFaucetInstance } from '../../../artifacts/ts'
+import { initTokensXY, withdrawTokens } from '../../../src/testUtils'
+import { FeeTier, PoolKey } from '../../../artifacts/ts/types'
+import { balanceOf, newFeeTier, newPoolKey } from '../../../src/utils'
+import { GlobalMaxTick, GlobalMinTick } from '../../../src'
+import { ChunkSize, ChunksPerBatch } from '../../../src/consts'
+import { getMaxChunk, getMaxTick, getMinTick, toLiquidity } from '../../../src/math'
+
+web3.setCurrentNodeProvider('http://127.0.0.1:22973')
+
+let invariant: Invariant
+let deployer: PrivateKeyWallet
+let positionOwner: PrivateKeyWallet
+let tokenX: TokenFaucetInstance
+let tokenY: TokenFaucetInstance
+let feeTier: FeeTier
+let poolKey: PoolKey
+
+describe('init invariant test', () => {
+  const initialFee = 0n
+  const [fee] = getBasicFeeTickSpacing()
+  const tickSpacing = 1n
+  const initSqrtPrice = 10n ** 24n
+  const supply = 10n ** 10n
+  const lowerTickIndex = GlobalMinTick
+  const upperTickIndex = GlobalMaxTick
+  const ticks = [-221818n, -221817n, -58n, 5n, 221817n, 221818n]
```

Comment:
it would better show intent by setting the first and last tick GlobalMinTick and GlobalMaxTick respectively